Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pake-cpace"
-version = "0.1.7"
+version = "0.2.0"
```

Comment:
When submitting a patch to a third-party project, please refrain from modifying the software version. This practice is not advisable. Instead, allow the project maintainers to decide when and what version to release.